rfcs/text/0017-remove-log-original.md

```diff
@@ -89,8 +95,12 @@ The following are the people that consulted on the contents of this RFC.
 
 ## References
 
-* [#841](https://github.com/elastic/ecs/issues/841)
-* [#777](https://github.com/elastic/integrations/issues/777)
+### Issues
+* [ecs #841](https://github.com/elastic/ecs/issues/841)
+* [integrations #777](https://github.com/elastic/integrations/issues/777)
+
+### Stack PRs
+* [kibana PR #102236](https://github.com/elastic/kibana/pull/102236)
 
 ### RFC Pull Requests
```

Comment:
Can you add the stage 3 PR here as well?

Reply:
Added stage 3 PR